src/vuex/getters.js

```diff
@@ -33,6 +33,10 @@ export const yourValidators = (state, getters) =>
           operator_address in getters.committedDelegations
       )
     : []
+export const validatorsWithRewards = (state, getters) =>
+  Object.entries(state.distribution.rewards)
+    .filter(([, rewards]) => rewards[getters.bondDenom] > 0)
+    .map(([validator]) => validator)
```

Comment:
I think you could leave off the last `.map`, since it's useful information that could be used elsewhere.